common/components/Checkbox/checkbox.module.scss

```diff
@@ -0,0 +1,46 @@
+@use "common/abstracts" as *;
+
+.task {
+  width: 100%;
+  --size: 2em;
+}
+
+.checkbox {
+  opacity: 0;
+  position: absolute;
+}
+
+.task label {
+  display: flex;
+  align-items: center;
+  flex-grow: 1;
+  justify-content: space-between;
+  cursor: pointer;
+}
+
+.customCheckbox {
+  position: relative;
+  display: inline-block;
+  width: var(--size);
+  height: var(--size);
+  border: 2px solid $grey;
+  border-radius: 50%;
```

Comment:
Up for discussion: 
Since we are allowing users to choose several options, should we change the form of the checkbox to square instead of circle? That's the most common look for checkboxes with multiple allowed answers. 

Could change to something like:
border-radius: 0.5rem; (line 27)
--size: 1.8em; (line 5)

<img width="451" alt="Skjermbilde 2024-08-02 kl  10 36 56" src="https://github.com/user-attachments/assets/451d661a-af56-4a67-b13d-ed5ada5abbc9">

Reply:
updated to squares.